### ISSUE TITLE
Fine-tune large vision language model for chat completion only

### ISSUE THREAD

**Issue by Liyan06** [huggingface/trl#2005]
How to fine-tune a large vision language model such as Llava on the generated prompts only? The current [code](https://github.com/huggingface/trl/blob/main/examples/scripts/vsft_llava.py) is fine-tuning on the full prompt but not on chat completion only like what previous `DataCollatorForCompletionOnlyLM` does for fine-tuning LLMs

**Comment by qgallouedec** [comment]
To finetune only on completions (exclude prompt) with SFT , you should set the prompt label to `-100`, as we do here 

https://github.com/huggingface/trl/blob/fc20db8873c058e82460166146b9590f03256f28/examples/scripts/vsft_llava.py#L123

for pad tokens

**Comment by Liyan06** [comment]
Thanks! This helps!
